src/index.ts

```diff
@@ -169,11 +172,16 @@ export class Driver {
     const { response, messages, index, status, toolCallIndex } = stage
     switch (status) {
       case 'pending': {
+        const tools = config.tools?.map(mcpxToolToAnthropic) || this.#tools
```

Comment:
:four: this would be the load-bearing change, right? (Assuming users could already pass `tool_choice` as a part of `config`?)